src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -559,7 +559,7 @@ public void incrementStatistic(@NotNull String statistic)
      * @param statistic the statistic.
      * @return amount of statistic.
      */
-    private int getStatisticAmount(@NotNull String statistic)
+    private int getStatisticAmount(@NotNull final String statistic)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this method 11, down to the maximum allowed 6. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)